data/config/items/equipment/leather-armour.json

```diff
@@ -1,4 +1,56 @@
 {
+    "rs:leather_cowl": {
+        "game_id": 1167,
+        "tradable": true,
+        "weight": 0.34,
+        "equippable": true,
+        "equipment_data": {
+            "equipment_slot": "feet",
+            "defensive_bonuses": {
+                "slash": 1,
+                "crush": 1
```

Comment:
Do we need to set these? It might be better to have no data compared to incorrect data.. I worry we might miss some in future when fixing them.

Reply:
Ill just fix them, I can't recall why they are set like that